src/lib/DepositController.js

```diff
@@ -9,29 +9,38 @@
 // Defines what happens when a button is clicked.
 
 import { setFormErrorsFromResponse } from './state/actions';
-import { PUBLISH_SUCCESS, SAVE_SUCCESS } from './state/types';
+import {
+  CREATE_DEPOSIT_SUCCESS,
+  PUBLISH_SUCCESS,
+  SAVE_SUCCESS,
+} from './state/types';
 
 export class DepositController {
-  constructor(apiClient) {
+  constructor(apiClient, fileUploader) {
     this.apiClient = apiClient;
+    this.fileUploader = fileUploader;
   }
 
   exists(record) {
```

Comment:
I would rename it to something more clear: `draftPreviouslyCreated` for example, it is not very clear when you use it otherwise: `if this.exist()...`

Reply:
I renamed it to `draftAlreadyCreated` :)